packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -1,8 +1,10 @@
-<div class="card">
-  <header class="hidden" id="fxa-connect-another-device-header"><h1 class="card-header mb-8">{{#t}}Connect another device{{/t}}</h1></header>
+<div class="card-base">
+  <header id="fxa-connect-another-device-header" class="{{^isFirefoxDesktop}}hidden{{/isFirefoxDesktop}}"><h1 class="text-base">{{#t}}Connect another device{{/t}}</h1></header>
```

Comment:
A few observations/questions:
- Spotted a style difference between the Figmas for Pair vs Password Reset/SMS - this heading is `bg-grey-400` in the other designs. We should prob align on one style.
- Could the wrapping `header` tag be removed, considering we already have a page level header with mozilla logo?

Reply:
> Spotted a style difference between the Figmas for Pair vs Password Reset/SMS - this heading is bg-grey-400 in the other designs. We should prob align on one style.

Hmm, good catch. I'll send a message in the channel for which we should go with, I imagine `text-grey-400`.

> Could the wrapping header tag be removed, considering we already have a page level header with mozilla logo?

Yes, I can change this to a `div`. Mostly, I just left it as-is because Backbone, but I believe your suggestion is probably "more" correct since this doesn't have a corresponding footer. 👍